features/step_definitions/is.rb

```diff
@@ -40,9 +40,19 @@
 end
 
 Given /^the(?: "([^"]*)")? image stream tag was created$/ do |istag_name|
-  @result = image_stream_tag(istag_name).wait_to_appear(user, 30)
+  org_project = project(generate: false) rescue nil
+
+  if istag_name
+    project_name, is_name = istag_name.split("/", 2)
+    unless is_name
+      is_name = project_name
+      project_name = nil
+    end
+    @result = image_stream_tag(is_name,project(project_name)).wait_to_appear(user, 30)
```

Comment:
The step name is `created`, but we only `wait_to_appear`.  Then where do we create them ?

Reply:
@liangxia new-app cmd will create imagestream if it not exist or check the openshift existing imagestream

Reply:
But several of your added steps happens before new-app. Where do we create them then ?

Reply:
They checked the openshift project imagestreams which are created by samples operator, but this pr need adjust, since samples operator has been an optional operator since 4.11 so it will not install always.
@kasturinarra @zhouying7780  I will remove the update for features/cli/oc_set_deployment_hook.feature file, please update later according to your change.
Thanks @liangxia